README.md

```diff
@@ -120,6 +120,55 @@ The `UserStatus.connections` (in-memory) collection contains information for all
 - `loginTime`: if authenticated, when the user logged in with this connection.
 - `idle`: `true` if idle monitoring is enabled on this connection and the client has gone idle.
 
+#### Usage with collection2
+
+If your project is using `aldeed:collection2` with a schema attached to `Meteor.users`, you need to add the following items to the schema to allow modifications of the status:
+
+```javascript
+import SimpleSchema from 'simpl-schema';
+
+const userSchema = new SimpleSchema({
+    status: {
+        type: Object,
+        optional: true,
+    },
+    'status.lastlogin': {
+        type: Object,
+        optional: true,
+    },
+    'status.lastlogin.date': {
+        type: Date,
+        optional: true,
+    },
+    'status.lastlogin.ipAddr': {
+        type: String,
+        optional: true,
+    },
+    'status.userAgent': {
+        type: String,
+        optional: true,
+    },
+    'status.iddle': {
```

Comment:
Shouldn't this be idle?

Reply:
Thank you, the field is indeed `idle`. I created a new PR to address this